main.css

```diff
@@ -0,0 +1,395 @@
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  position: fixed;
+  z-index: 9999;
+  width: 1440px;
+  height: 65px;
+  border: 1px solid #344563;
+  margin-top: -8;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.link-div {
+  width: 100%;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  margin-top: 25px;
+  margin-right: 100px;
+}
+
+span a {
+  color: #6070ff;
+}
+
+.link-div a {
+  float: right;
+  margin-right: 30px;
+}
+
+span {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  height: 20px;
+  width: 76px;
+  margin-left: 100px;
+  margin-top: 25px;
+}
+
+.page1 {
+  display: flex;
+  flex-direction: column;
+  height: 916px;
+  width: 1440px;
+  background-color: white;
+  border-bottom-left-radius: 100px;
+  background-image: url(assets/images/Header_bg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.middle {
+  width: 1440px;
+}
+
+.h1 {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 110px;
+  margin: 250px 260px 370px 260px;
+  color: #091e42;
+  letter-spacing: 0.37px;
+  height: 122px;
+  width: 920px;
+}
+
+#p1 {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  color: #344563;
+}
+
+.P2 {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  line-height: 24px;
+}
+
+.p3 {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+.soc_img1 {
+  margin-left: 260px;
+}
+
+.cards {
+  display: flex;
+  background-color: white;
+  margin: 142px;
+  height: 496px;
+  width: 1156px;
+  border-radius: 8px;
+}
+
+.left_align {
+  margin: 24px 0 24px 24px;
+}
+
+.right_align {
+  margin: 24px 22px 24px -4px;
+}
+
+.img-txt1 {
+  margin: 48px 46px 24px 46px;
+}
+
+.img-txt2 {
+  margin: 48px;
+}
+
+.btns {
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  margin-top: 56px;
+  border: 1px solid #396df2;
+  padding: 12px;
+  border-radius: 10px;
+}
+
+.resume_btn {
+  margin-top: 34px;
+}
+
+.btns:hover {
+  height: 48px;
+  width: 124px;
+  background-color: #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  border-width: 1px;
+  gap: 10px;
+  color: white;
+}
+
+.btns:active {
+  height: 48px;
+  width: 124px;
+  background-color: #2230d2;
+  border-radius: 8px;
+  padding: 12px;
+  border-width: 1px;
+  gap: 10px;
+  color: white;
+}
+
+.resume_btn:active {
+  height: 48px;
+  width: 162px;
+  background-color: #2230d2;
+  border-radius: 8px;
+  padding: 12px;
+  border-width: 1px;
+  gap: 10px;
+  color: white;
+}
+
+.resume_btn:hover {
+  width: 162px;
+}
+
+.tags {
+  margin-top: 24px;
+}
+
+.last_page {
+  display: flex;
+  background-color: #fff;
+  width: 1440px;
+  height: 924px;
+  border-top-right-radius: 100px;
+  padding-top: 142px;
+  margin-bottom: -320px;
+}
+
+.left_block {
+  width: 447px;
+  height: 392px;
+  margin: 0 0 0 142px;
+}
```

Comment:
great design @Bludidi 🤝

- [ ] Kindly make sure your font color is uniform throughout the design. Kindly fix the font color of this section.
- Your design
![Your contact](https://user-images.githubusercontent.com/63917725/187960997-4ae9cb70-3f74-4f80-be45-83b067ed7baa.PNG)

- Figma design
![Figma design](https://user-images.githubusercontent.com/63917725/187961036-17bf3869-113b-4450-a447-e8c4844c093f.PNG)